packages/stack/src/lib/stack-app.ts

```diff
@@ -778,6 +778,20 @@ class _StackClientAppImpl<HasTokenStore extends boolean, ProjectId extends strin
     }
     return errorCode;
   }
+ 
+  async createUser(options: {
```

Comment:
There is some code duplication between this and signUpWithCredential now. Can we make the latter call this function?